__tests__/puppeteer.js

```diff
@@ -0,0 +1,39 @@
+const puppeteer = require('puppeteer');
+const APP = `http://localhost:8080/`;
+
+
+describe('Front-end Integration/Features', () => {
+  let browser;
+  let page;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      args: ['--no-sandbox', '--disable-setupid-sandbox'],
+    });
+    page = await browser.newPage();
+  });
+
+  afterAll(() => {
+    browser.close();
+  });
+
+  describe('Initial display', () => {
+    it('loads successfully', async () => {
+      // We navigate to the page at the beginning of each case so we have a
+      // fresh start
+      await page.goto(APP);
+      await page.waitForSelector('#body');
+      const title = await page.$eval('#body', el => el.innerHTML);
+      expect(title).toBe('');
```

Comment:
Is the title empty? Wouldn't this just fail since we have a title in `index.html`?

Reply:
Is that a problem? I wanted to get the basic template of the tests together first before building out specific passing tests.

Reply:
ah ok, it wasn't clear. Might be worth adding a comment in if that's your intention 😃